importlab/resolve.py

```diff
@@ -35,6 +35,14 @@ def __init__(self, path, module_name):
     def is_extension(self):
         return self.path.endswith('.so')
 
+    @property
+    def package_name(self):
+        f, _ = os.path.splitext(self.path)
+        if f.endswith('__init__'):
+            return self.module_name
+        else:
+            return self.module_name[:self.module_name.rfind('.')]
+
     @property
     def short_path(self):
         # TODO: We really need to know the module name of the including file
```

Comment:
Does this TODO still apply? The short paths for relative imports look correct to me now.